latex/blackarch-guide-de.tex

```diff
@@ -31,7 +31,8 @@
 \widowpenalty10000
 \displaywidowpenalty=10000
 \usepackage{ltablex}
-
\keepXColumns
```

Comment:
Is that a fix or a mistake?

Reply:
that's weird... I can't find where did I change this in my local repo... (but it seems like it shouldn't be there)

Reply:
after some investigation, yes, it should be removed. Sorry for accidentally squishing it into one commit that has nothing to do with it. Should I split the commit into two?

Reply:
Nop that's good then.